skopt/optimizer/optimizer.py

```diff
@@ -136,7 +136,7 @@ class Optimizer(object):
     """
     def __init__(self, dimensions, base_estimator="gp",
                  n_random_starts=None, n_initial_points=10,
-                 acq_func="gp_hedge",
+                 n_jobs=1, acq_func="gp_hedge",
```

Comment:
Please add description of `n_jobs` to docstrings.

Reply:
+1